knowledge-base/pickers-readonly-hidden-input.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Disable Typing in DatePicker Input and Always Use the Calendar 
+description: How to disable typing in the Blazor date/time pickers and force users to always use the popup Calendar and TimeView.
+type: how-to
+page_title: Hide or Disable Typing in the DatePicker Textbox
+slug: pickers-kb-readonly-dateinput
+position: 
+tags: telerik, blazor, datepicker, datetimepicker, timepicker, readonly
+ticketid: 1597101, 1619245
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>
+                DatePicker for Blazor, <br />
+                DateTimePicker for Blazor, <br />
+                TimePicker for Blazor
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to disable direct typing in the DatePicker and TimePicker components? I want to restrict user to use only the Calendar and TimeView popups.
```

Comment:
How about using a bullet list for the different forms of the question? This will add a clear structure to the description.

```suggestion
This how-to article answers the following questions:

* How to disable direct typing in the DatePicker and TimePicker components? I want to restrict users to use only the Calendar and TimeView popups.
```